spanner/spanner_quickstart/main.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // Sample spanner_quickstart is a basic program that uses Cloud Spanner.
+// [START spanner_quickstart]
```

Comment:
Please move this above the doc comment, or below it. As it is, it'll be included in package docs.

Reply:
Done